### ISSUE TITLE
How to use at `@` character?

### ISSUE THREAD

**Issue by hugotox** [pablo-abc/svelte-markdown#87]
If I try to use the at character, it renders a big string of bad characters. Any idea how to escape it?

e.g. 

```html
<SvelteMarkdown source="hello@mail.com" />
```

Renders:
```
[&#x68;&#101;&#108;&#x6c;&#x6f;&#64;&#x6d;&#97;&#105;&#x6c;&#x2e;&#x63;&#111;&#109;](mailto:&#x68;&#101;&#108;&#x6c;&#x6f;&#64;&#x6d;&#97;&#105;&#x6c;&#x2e;&#x63;&#111;&#109;)
```

**Comment by domhel** [comment]
According to
this https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses
and this https://github.com/markedjs/marked/issues/2397
it should only be a link when wrapped with <...> but it just doesn't work.
I now use backticks so it's treated as code but I'm not happy

**Comment by hugotox** [comment]
Links work, just the `@` character alone fails. See my example here https://stackblitz.com/edit/sveltejs-kit-template-default-burzvt?file=src%2Froutes%2F%2Bpage.svelte

**Comment by williamoverton** [comment]
@hugotox did you manage to get around this?

**Comment by hugotox** [comment]
> @hugotox did you manage to get around this?

Yes, kindof. The character `@` as it is doesn't work, but you can use the html entity `&#64;`. Also if rendering an email address then it works fine, e.g. `[info@domain.com](mailto:info@domain.com)`

**Comment by OrkhanAlikhanov** [comment]
Ran into the same issue. Solution is mentioned here: https://github.com/pablo-abc/svelte-markdown/issues/50

**Comment by dreamscached** [comment]
`mangle: false` doesn't work for me, how does anyone manage to unescape HTML entities?

**Comment by emiliedebra** [comment]
`mangle: false` does not seem to be working anymore, does anyone know what the new way of applying this is?
